### PR TITLE
Fixed #25127 -- Add docs on organizing models

### PR DESCRIPTION
**Pull request by palmerev**
Added a section to the Models docs on organizing models across multiple files. Example recommends
explicit imports over 'import *'.

### CODE REVIEW

**Comment by charettes** [comment]
Thanks for your contribution. Is there a reason not to suggest using a `models` package with `robot` and `person` submodules instead? It seems to be the approach suggested in the ticket and it's a more common pattern IMHO.

**Comment by palmerev** [comment]
That's essentially what the example does, but I didn't use the "package" and "submodule" terminology. Or do you mean making submodules as their own directories?

**Comment by charettes** [comment]
Ohh never mind, I wrongly assumed you were suffixing the `models` submodules with `_models` because they were not part of the `models` package. I think I got confused because I usually don't use such a suffix. (e.g. `models/robot.py` instead `models/robot_models.py`).

**Comment by palmerev** [comment]
If it's confusing, then maybe I should remove the `_models` suffix.

**Comment by charettes** [comment]
Let's wait to get more feedback before performing any changes. It might only be me.

File: docs/topics/db/models.txt [hunk 1]
```diff
@@ -1415,3 +1415,47 @@ any model field in any ancestor model.
...
+Organizing models
+=================
+
+The :djadmin:`manage.py startapp <startapp>` script creates a package structure
+for an application that includes a file called ``models.py``, but you may
+also define multiple files containing models, contained in a module. For example,
+you can remove the ``models.py`` and create a ``myproject/myapp/models/`` with
+two files (besides an ``__init__.py``), ``person_models.py`` and ``robot_models.py``:
+
+``person_models.py``::
```
Comment (timgraham) [inline docs/topics/db/models.txt]: You can use `.. snippet::` with `:filename:` instead, however, I don't think it's important to show these two files. Defining models is already covered.
As Simon suggested, I'd drop the `_models` suffix.

File: docs/topics/db/models.txt [hunk 2]
```diff
@@ -1415,3 +1415,47 @@ any model field in any ancestor model.
...
+    from django.db import models
+
+    class Robot(models.Model):
+        model_number = models.CharField(max_length=30)
+        unit_number = models.CharField(max_length=30)
+
+
+Then import your model classes into ``models/__init__.py``. It is recommended
+to explicitly import each model class, rather than using
+``from .models import *``.
```
Comment (timgraham) [inline docs/topics/db/models.txt]: which will... (explain bad things about using import *?)

File: docs/topics/db/models.txt [hunk 3]
```diff
@@ -1415,3 +1415,47 @@ any model field in any ancestor model.
...
+
+    from .person_models import Person
+    from .robot_models import Robot
+
+You can then import classes directly from the ``models`` module as you normally
+would.
+
+``myapp/views.py``::
+
+    from .models import Person
```
Comment (timgraham) [inline docs/topics/db/models.txt]: Not sure we need to show how to import models normally. I don't think someone would get this far in the docs without understanding how that works.

File: docs/topics/db/models.txt [hunk 4]
```diff
@@ -1415,3 +1415,47 @@ any model field in any ancestor model.
     :doc:`The Models Reference </ref/models/index>`
         Covers all the model related APIs including model fields, related
         objects, and ``QuerySet``.
+
+Organizing models
```
Comment (timgraham) [inline docs/topics/db/models.txt]: This section should go above the seealso which is meant for the bottom of the document.

File: docs/topics/db/models.txt [hunk 5]
```diff
@@ -1415,3 +1415,47 @@ any model field in any ancestor model.
...
+=================
+
+The :djadmin:`manage.py startapp <startapp>` script creates a package structure
+for an application that includes a file called ``models.py``, but you may
+also define multiple files containing models, contained in a module. For example,
```
Comment (keturn) [inline docs/topics/db/models.txt]: I think the word for the-thing-those-multiple-files-are-contained-in is "package." (or maybe "sub-package"?)

**Comment by carljm** [comment]
FWIW, I also would prefer `models/robot.py` over `/models/robot_models.py`.

**Comment by palmerev** [comment]
I just pushed some changes that should address the issues you mentioned.

**Comment by timgraham** [comment]
merged in 84d8d1d7151a4ee70ae35037d37f76a40d18da64, thanks!

**Pull request closed without merging**